doc/articles/faq.md

```diff
@@ -286,3 +286,83 @@ Yes, make sure to use the following project definition:
     <Reference Include="System.Runtime.Serialization" />
 </ItemGroup>
 ```
+
+## Does Uno offer a TreeView control?
+
+It's in the UWP API, but [not implemented yet](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tree-view)
+
+## Is there a table of all UWP controls and their features compared to what's offered under Uno?
+
+https://github.com/nventive/Uno/blob/master/doc/articles/supported-features.md
+
+##  Is there an Uno template that is based on portable class library?
+
+https://github.com/nventive/Uno/pull/132
+
+## Is there a Visual Studio template for Uno that incorporates the Prism library?
+
+No, but this sample can serve as a base: https://github.com/nventive/Uno.Prism/tree/uno/Sandbox/Windows10/HelloWorld
+
+## I get errors when serializing Json in Uno Wasm
+
+If you are using JSON.NET, you need [this](https://github.com/nventive/Uno.Playground/blob/master/src/Uno.Playground.WASM/LinkerConfig.xml) 
+This file is referenced in the .csproj like [that](https://github.com/nventive/Uno.Playground/blob/master/src/Uno.Playground.WASM/Uno.Playground.WASM.csproj#L43)
+
+## Is NavigationView supported in Uno?
+
+It is indeed not implemented at the time, see [nventive/Uno#4](https://github.com/nventive/Uno/issues/4) for now
+
+## Is there any particular reason that Uno uses a shared project? and is it possible to use a netstandard project instead?
+
+The view layer needs to be in a shared project because it has native dependencies.
+For your view models and business logic, it's fine to use a separate netstandard project.
+
+## Are there any Visual Studio project creation templates for Uno yet?
+
+Yes. Here are the [templates].(https://marketplace.visualstudio.com/items?itemName=nventivecorp.uno-platform-addin)
+
+## How do I add the NuGet package if there's no Packages node for Shared Projects?
+
+Go to 'Manage NuGet packages for solution...'
+Find the ReactiveUI package
+Select all your platform heads (.Wasm, .Android etc) and hit Install
+
+## Is RichEditBox supported in Uno.Platform?
+
+Not yet.
+
+## Is there a way to use local css/js libraries and not those on a CDN?
+
+you can specify a custom HTML template like [this](https://github.com/nventive/Uno.Wasm.Bootstrap#indexhtml-content-override)
+
+## Debugging a published nuget package breaks the Xamarin iOS and Android debugger
+
+Please see [here](https://developercommunity.visualstudio.com/content/problem/306764/debugging-a-published-nuget-package-break-the-xama.html)
+
+## Does Uno offer an `AutoSuggestBox`?
+
+[Yes](https://github.com/nventive/Uno/blob/master/src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs)
+
+## Is there a database that works on all platforms?
+
+SQLite should work on all platforms (except Wasm) https://www.nuget.org/packages/sqlite-net-pcl/
```

Comment:
Change this, _Yes, and you can use this WebAssembly SQLite provider:  https://github.com/nventive/Uno.SQLitePCLRaw.Wasm_